virttest/qemu_storage.py

```diff
@@ -44,7 +44,7 @@ def filename_to_file_opts(filename):
                              'transport': 'tcp',
                              'portal': matches.group('portal'),
                              'target': matches.group('target'),
-                             'lun': matches.group('lun')}
+                             'lun': int(matches.group('lun'))}
```

Comment:
iscsi image's json format repr in qemu-img command line requires an integer value of lun instead of a string. For qemu-kvm, both are OK, so here change it to int.